### ISSUE TITLE
DB/Creature: Kristen Dipswitch

### ISSUE THREAD

**Issue by Killyana** [TrinityCore/TrinityCore#24940]
**Description:** https://www.wowhead.com/npc=18294/kristen-dipswitch#starts must off the quest https://www.wowhead.com/quest=9897/im-saved only once you kill https://www.wowhead.com/npc=18297/gankly-rottenfist#comments and save her.
" Kristen Dipswitch arrives on a Mechanostrider from the Mainpath near the Camp. When she dismounts Gankly Rottenfist spawns near the Tent an starts to attack her. If you Kill him, she will offer I'm Saved!"

**Branch(es):** Both

**Comment by offl** [comment]
https://github.com/cmangos/tbc-db/commit/1a6c3ffc105990cec95d8018412cce086bb41ecc https://github.com/cmangos/tbc-db/commit/d5783ad64a19c95565fce7dca3cfe8cbb49d6096

**Comment by illfated** [comment]
Not sure if I remember this exactly, but is it part of the issue that she currently offers the quest directly when you meet her on the Nagrand side of the border with Zangarmarsh?

**Comment by offl** [comment]
Yes.

**Comment by offl** [comment]
Duplicate of #24983
